### ISSUE TITLE
Backups in offline mode

### ISSUE THREAD

**Issue by kingsleyzissou** [ComplianceAsCode/content#10926]
#### Share the context

We are currently updating our osbuild stage tests [1] and ran into an issue with the OpenSCAP remediation tests. The test basically creates an image with remediations and one without and compares the diff of the two images. We noticed that `authselect` was creating backups.

#### Description of problem:

The OpenSCAP offline remediation created backups for a freshly created image. 

OS:
`Fedora-38`

Version:
`scap-security-guide-0.1.68-1.fc38`

#### Proposed change:

Backups for an already running system makes sense, however, we were wondering if it makes sense to enable these for a freshly created image? Would it be possible to disable the backups in offline mode?

#### References:

1. https://github.com/osbuild/osbuild/pull/1351

**Comment by marcusburghardt** [comment]
Yes, we can update the macro to make the backup optional or even based in a conditional. I will take a look on this.

**Comment by ggbecker** [comment]
> Yes, we can update the macro to make the backup optional or even based in a conditional. I will take a look on this.

The question is, how are you going to control this via the remediation. I don't think we can do it just with the fact that the scanning runs on offline mode.

**Comment by marcusburghardt** [comment]
> > Yes, we can update the macro to make the backup optional or even based in a conditional. I will take a look on this.
> 
> The question is, how are you going to control this via the remediation. I don't think we can do it just with the fact that the scanning runs on offline mode.

I don't know details now, but it is something I expect to figure out when investigating the case.

**Comment by marcusburghardt** [comment]
Related: https://github.com/ComplianceAsCode/content/issues/10967
